pkg/node/heartbeat/types.go

```diff
@@ -0,0 +1,21 @@
+package heartbeat
+
+import "time"
+
+const (
+	heartbeatTopic               = "heartbeat"
+	heartbeatQueueCheckFrequency = 5 * time.Second
+
+	unhealthyAfter = 30 * time.Second
+	unknownAfter   = 60 * time.Second
+)
```

Comment:
In a similar vein to my comment on the ManagementClient, I think these should be configurable, with defaults that don't require the ManagementClient to heartbeat twice as frequently as the configured heartbeat interval.